scripts/detect_arch

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [ -n "$(uname -m | grep arm64)" ] || [ -n "$(uname -m | grep aarch64)" ]; then
+	architecture="arm64"
+elif [ -n "$(uname -m | grep 64)" ]; then
+	architecture="x86-64"
+elif [ -n "$(uname -m | grep 86)" ]; then
+	architecture="i386"
+elif [ -n "$(uname -m | grep arm)" ]; then
+	architecture="arm"
+else
+	ynh_die --message="Unable to detect your achitecture, please open a bug describing \
+        your hardware and the result of the command \"uname -m\"." 1
+fi
```

Comment:
Eh, wouldn't it be better to have this as a helper that returns the detected arch instead, and use it like `architecture=$(ynh_detect_arch)` for example ? 

Also note that you are using `ynh_die` in there, but later source this file *before* the yunohost helpers, so probably it would crash miserably because it doesn't know `ynh_die` yet (in the case where it want to run `ynh_die`...)

Reply:
Well, that's an actual experimental helper : https://github.com/YunoHost-Apps/Experimental_helpers/blob/master/detect_arch/detect_arch

But I'm OK, I will make a PR to change it to be able to have `architecture=$(ynh_detect_arch)`

I let you know when it's done

Reply:
Replaced by new helper ynh_detect_arch__2